base/src/main/java/com/smartdevicelink/session/ISdlSessionListener.java

```diff
@@ -82,4 +83,6 @@
      * @param sessionID the session ID that this auth token is associated with
      */
     void onAuthTokenReceived(String authToken, int sessionID);
+
+    boolean onVehicleTypeReceived(VehicleType type, String systemSoftwareVersion, String systemHardwareVersion);
```

Comment:
@kostyaBoss On line 87 does this method `onVehicleTypeReceived` need JavaDoc comments around it like the other methods?

Reply:
@santanamk, I don't have the valid one, that's why I didn't add anything